_benchmark/installing-benchmark.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: Installing OpenSearch Benchmark
+nav_order: 5
+has_children: false
+---
+
+# Installing OpenSearch Benchmark
+
+You can install OpenSearch Benchmark directly on a host running Linux or macOS or you can run OpenSearch Benchmark in a Docker container on any compatible host. This page describes high-level considerations for your OpenSearch Benchmark host, as well as instructions for installing OpenSearch Benchmark.
+
+
+## Choosing appropriate hardware
+
+OpenSearch Benchmark can be used to provision OpenSearch nodes for testing. If you intend to use OpenSearch Benchmark to provision nodes in your environment, then install OpenSearch Benchmark directly on each host in the cluster. Additionally, you must configure each host in the cluster for OpenSearch. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/) for guidance on important host settings you should review.
```

Comment:
The third sentence is slightly vague. Do we mean something like "in the cluster to work with OpenSearch"?